### PR TITLE
Meshmodel Postgres & Kubernetes Component

### LINKED ISSUE #8758: [meshmodels] [UX] : Icon for Meshmodel Postgres Component - Postgres Cluster
### Current Behavior
Meshery is a powerful platform that contains over [200 integrations](https://layer5.io/cloud-native-management/meshery/integrations) for managing cloud-native applications. MeshMap is an extension of Meshery that visually represents these integrations and components. Currently, most of the icons used in MeshMap have the default shape of a circle. However, to improve visual clarity and consistency, we need to design custom icons for MeshMap components.
### Desired Behavior
A new Icon for  **Postgres Cluster** needs to be created. This issue calls for a volunteer to draft a new Icon.
### Instructions for Components
While the default shape for new components is a circle, each component should be considered for its best-fit shape.
1. Review and familiarize with the available set of predefined relationship types. Refer to the Cytoscape [node types](https://js.cytoscape.org/demos/node-types/) for a list of possible shapes.
1. Propose a specific shape, best-suited to visually represent the Component. Example - Deployment as a pentagon, DaemonSet as a skull icon etc.
3. Create two versions of the icon one is white and another one is colored
4.  Make a PR after adding the icon inside the [new-icon folder](https://github.com/meshery/meshery/tree/master/ui/assets/new-icons) present in the meshery repository
5. Provide the icon in SVG format

### PR DESCRIPTION
**Pull request by ptdatta**
**Notes for Reviewers**

This PR fixes 
#8758 
#8757 

- For the Kubernetes icon we can use Cytoscape `round-heptagon` shape.
- For the Postgres icon we can use Cytoscape `vee` or reverse `round-triangle` shape. 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by ritiksaxena124** [comment]
The icon needs to be created of component, not of model Postgres and Kubernetes are models. So you will have to create an Icon for the Postgres Cluster and Kubernetes Endpoints

**Comment by ptdatta** [comment]
Thanks @ritiksaxena124, does I have to create something like: 
image
for postgres and 
image
for kubernetes

**Comment by csengineer1990** [comment]
> Thanks @ritiksaxena124, does I have to create something like: image for Postgres and image for Kubernetes ???


@ptdatta could you please tell me which tool are using to create icons? I also want to contribute.

**Comment by saurabh100ni** [comment]
Let's discuss this on the Meshery dev call tomorrow,
Please add this as your agenda item in Docs: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit

**Comment by ptdatta** [comment]
@ritiksaxena124 Is it now correct?

**Review by leecalcote — APPROVED** [review]
Good work, @ptdatta. 👍

File: project.inlang.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (leecalcote) [inline project.inlang.json:1]: Please remove this file

File: ui/assets/new-icons/Kubernetes-white.svg
Comment (leecalcote) [inline ui/assets/new-icons/Kubernetes-white.svg:1]: Please deliver colored and white versions of just the icons and not the shapes.

Note, that we already have the Postgres elephant, so, you'll need to bring something else.

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: project.inlang.json (hunk 1 above)
Comment (leecalcote) [reply project.inlang.json:1]: @ptdatta are you going to finish this?

Comment (ptdatta) [reply project.inlang.json:1]: I thought it's already completed.

Comment (leecalcote) [reply project.inlang.json:1]: @ptdatta you thing that you have removed this file?

**Review by leecalcote — APPROVED** [review]

**Pull request merged**